Corona-Warn-App/src/main/res/navigation/nav_graph.xml

```diff
@@ -49,6 +49,9 @@
         app:destination="@id/submissionTestResultPendingFragment"
         app:popUpTo="@id/mainFragment"
         app:popUpToInclusive="false" />
+    <action
+        android:id="@+id/action_global_to_rampdownNoticeFragment"
+        app:destination="@id/rampdownNoticeFragment" />
```

Comment:
why global action ,it should be only for `HomeFragment` ?

Reply:
Added to status_nav_graph